### ISSUE TITLE
imagenet training

### ISSUE THREAD

**Issue by YunYunY** [bytedance/TWIST#3]
Hi,
thanks for sharing your work. Did you do any preprocessing of imagenet? Or just setting data-path as the raw imagenet is enough? 
Thank you.

**Comment by WangFeng18** [comment]
No, we do not modify any content of images in ImageNet, the format of raw imagenet folders should work.
